posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -0,0 +1,268 @@
+---
+layout: post
+title: "Jakarta EE 10 Core Profile makes it debut in Open Liberty 22.0.0.9-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/reece-nana
+author_github: https://github.com/reece-nana
+seo-title: Jakarta EE 10 Core Profile makes it debut in Open Liberty 22.0.0.9-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.9-beta brings you the hot off the press Jakarta EE 10 Core Profile. Jakarta EE 10 is the first release to provide updates to the Jakarta EE specifications since Java EE 8 in 2017.  MicroProfile OpenAPI 3.1 and Password Utilities 1.1 are also made available in this beta release.
+blog_description: Open Liberty 22.0.0.9-beta brings you the hot off the press Jakarta EE 10 Core Profile. Jakarta EE 10 is the first release to provide updates to the Jakarta EE specifications since Java EE 8 in 2017.  MicroProfile OpenAPI 3.1 and Password Utilities 1.1 are also made available in this beta release.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Jakarta EE 10 Core Profile makes it debut in Open Liberty 22.0.0.9-beta
+Reece Nana <https://github.com/reece-nana>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+Open Liberty 22.0.0.9-beta brings you the hot off the press Jakarta EE 10 Core Profile. Jakarta EE 10 is the first release that provides some updates to the Jakarta EE specifications since Java EE 8 in 2017. While Jakarta EE 10 release is around the corner, Open Liberty has consumed many of the Jakarta EE 10 specifications and put them in the 22.0.0.9-beta.  In this beta release, the features that make up the new Jakarta EE 10 Core Profile are made available: 
+
+* Jakarta RESTful Web Services 3.1: support multipart/form-data.
+* Jakarta JSON-B 3.0: deserialization of null to JsonValue.NULL, default PropertyVisibilityStrategy Implemenation etc.
+* Jakarta JSON-P 2.1: handling duplicated keys; defined more interfaces for primitives, lists and objects 
+* Jakarta CDI 4.0: defining new extension APIs; removing reflections; adding new reflection-free language model; empty beans.xml means that only annotated classes are beans; removal of deprecated APIs
+** Jakarta Annotation 2.1: allowing `@Priority` to be used on any program elements; adding `@Nullable` and `@NotNull`
```

Comment:
```suggestion
* Jakarta Annotation 2.1: allowing `@Priority` to be used on any program elements; adding `@Nullable` and `@NotNull`
```

Reply:
I purposefully nested annotation and interceptor under CDI, since, to my knowledge, that's how they're made available in Liberty (aka, you can't enable those independently).

Reply:
```suggestion
** Jakarta Annotation 2.1: allowing `@Priority` to be used on any program element; adding `@Nullable` and `@NotNull`
```